code/modules/mob/living/simple_animal/hostile/thing.dm

```diff
@@ -0,0 +1,1381 @@
+				anchored = 1  // Point of no return, both the thing and the target are anchored and the darkness quickly forms.
+				H.anchored = 1
+				flick("thing_absorbin2",src)
+				sleep(10.25)
+				icon_state = "thing_absorbing"
+				var/absorbckey = H.ckey
+				sleep(29.4) // Bit of time to damage the thing. Enough time to play the animation six times.
+				H.add_splatter_floor()
+				qdel(absorb_target)
+				if(can_convert == 1)
+					var/turf/B = get_turf(src)
+					var/atom/movable/overlay/animation = new /atom/movable/overlay(B)
+					animation.name = "???"
+					animation.density = 0
+					animation.anchored = 1
+					animation.icon = 'icons/mob/mob.dmi'
+					animation.icon_state = "serf_appear"
+					animation.layer = 4
+					animation.desc = "What..."
+					animation.master = holder
+					flick("serf_appear",animation)
+					sleep(14)
+					if(animation)
+						qdel(animation)
+						var/mob/living/simple_animal/hostile/serf/S = new /mob/living/simple_animal/hostile/serf(loc)
+						S.ckey = absorbckey
+						to_chat(S, "<span class='danger'>You are now a chaos bringer! Follow your master to victory. The longer you survive, the stronger you become.</span>")
+						S.dir = 2
+						S.visible_message("<B>A strange being rises...</b>")
+				flick("thing_absorbout",src)
+				playsound(src, 'sound/thing/absorb.ogg', 80, 1)
+				visible_message("<B>The tentacles are absorbed in a sickening crunch, and so is \the [H]...</b>")
+				layer = 4.1
+				sleep(11.8)
+				dir = 1
+				icon_state = "thing"
+				anchored = 0
+				canmove = 1
+				absorb_target = null
+				absorbckey = null
+				absorbing = 0
+				points++ // Get a point
+				absorptions++
+				layer = 4
+				if(health < maxHealth)
+					if(health == 0)
+						adjustBruteLoss(-(maxHealth/10)) // The absorption can't be stopped. The thing will be revived if it dies, although it will have very low health.
+						stat = 0
+					else
+						adjustBruteLoss(-(maxHealth/4)) // If the thing was not killed during the absorption it will gain a good amount of health.
+				if (health > maxHealth)
+					health = maxHealth // Visual fix for the health indicator saying you have higher than 100% health when it reality it still counts as 100%.
+			else // Hurrah! The target was saved!
+				if(health > 0) // Thing is still alive. Let's make it able to run away and think of its actions.
+					dir = 1
+					icon_state = "thing"
+					anchored = 0
+					canmove = 1
+					absorb_target = null
+					absorbing = 0
+					visible_message("<span class='warning'>\the [src] was interrupted!</span>")
+					playsound(src, 'sound/thing/interrupted.ogg', 80, 1)
+					deathmessage = "crumbles into dust!"
+				else // The thing was killed and it or its target was pulled away. Pulling is a vital thing to do.
+					icon_state = "thing_dead"
+					visible_message("<span class='danger'>NO!</span>")
+					anchored = 0
+					absorbing = 0
+
+/mob/living/simple_animal/hostile/thing/verb/Laycreep()
```

Comment:
I think it might be a better idea to change all these ability verbs into action buttons that are granted when the ability is purchased - For one, we want to focus on using action buttons, and second, we want to avoid overwhelming players with a menagerie of buttons they can't use yet.